mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/components/tabs/automation_flow/index.tsx

```diff
@@ -59,5 +61,29 @@ export function AutomationFlow(): JSX.Element {
 
   const isLoading = section.data === undefined;
 
-  return !isLoading ? <Automation context="view" /> : <AutomationPlaceholder />;
+  if (isLoading) {
+    return <AutomationPlaceholder />;
+  }
+
+  return (
+    <>
+      {section.data.tree_is_inconsistent && (
+        <div className="mailpoet-automation-editor-automation-notices">
+          <Notice
+            status="warning"
+            isDismissible={false}
+            className="mailpoet-automation-flow-notice"
+          >
+            <p>
+              {__(
+                'In this time period, the automation structure did change and therefore some numbers in the flow chart might not be accurate.',
```

Comment:
This message doesn't tell me much about what the inaccuracies might be, and doesn't provide any guidance for how to view stats that are accurate. Can we explain what the inaccuracies would be? Maybe we could include in this message a list of the dates of each version, so a user could limit their search to when just a single version was active.

Reply:
Created a ticket to discuss this further: [MAILPOET-5461]